stories/Thing.stories.tsx

```diff
@@ -158,5 +158,24 @@ export const _CustomFont = ({font, ...args}: CustomFontProps) => {
 export const CustomFont = _CustomFont.bind({});
 
 CustomFont.args = {
-  font: 'Roboto'
-}
+  font: 'Roboto',
+};
+
+export const WithLengthLimit = () => {
+  const [inValue, setInValue] = useState<string>('Has a limit = to 20!');
```

Comment:
It would be cool if we could set this value with the storybook addons. I will create another ticket for that though.